ynr/apps/parties/tests/test_importer.py

```diff
@@ -102,7 +115,9 @@ def test_cleaned_name(self):
 
     @patch("parties.importer.ECEmblem.download_emblem")
     def test_save(self, FakeEmblemPath):
-        FakeEmblemPath.return_value = EXAMPLE_IMAGE_FILENAME
+        FakeEmblemPath.return_value = make_tmp_file_from_source(
+            EXAMPLE_IMAGE_FILENAME
+        )
```

Comment:
is this another place where we need to clean up our temp files when we're done?

Reply:
Nope, this is needed _because_ of cleaning up after an import – a patched `download_emblem` to return the test fixture image (rather than downloading one), but then the clean up code deleted the test image. This is a change that copies that test image to a temp location that then gets deleted at the end of the test.

Reply:
sorted :+1: